src/lxc/conf.c

```diff
@@ -2645,6 +2645,7 @@ struct lxc_conf *lxc_conf_init(void)
 		free(new);
 		return NULL;
 	}
+	new->rootfs.managed = true;
```

Comment:
Shouldn't `lxc_conf_init()` set this to the same as the `clear` method?

Reply:
The other way around actually. The default should be that it's managed. So clear should set to `true` most likely.

Reply:
Unless that is to confusing?

Reply:
*too

Reply:
Whichever is less of a change to the previous behavior I guess?

Reply:
`lxc.rootfs.managed` is supposed to mean "the rootfs is managed by this lxc instance" so the default should be `true`.

Reply:
Would calling it `lxc.rootfs.external` and reversing the logic make it less confusing maybe?

Reply:
Yeah, but I like the `managed` part and `external` is harder to grasp. Usually the `lxc.rootfs.managed` config key will not be set anyway. It will only be set when set to `false`.